src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -36,17 +38,22 @@
     public EditCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS,
-                    PREFIX_RISKAPPETITE, PREFIX_DISPOSABLEINCOME, PREFIX_CURRENTPLAN, PREFIX_LASTMET, PREFIX_TAG);
-
+                ArgumentTokenizer.tokenize(args, PREFIX_CLIENTID, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
+                        PREFIX_ADDRESS, PREFIX_RISKAPPETITE, PREFIX_DISPOSABLEINCOME, PREFIX_CURRENTPLAN,
+                        PREFIX_LASTMET, PREFIX_TAG);
+        ClientId clientId;
         Index index;
-
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
+            clientId = new ClientId(Integer.toString(index.getOneBased()));
```

Comment:
Should it be this instead?
```suggestion
            clientId = ParserUtil.parseClientId(argMultimap.getPreamble());
```